libtmux/test.py

```diff
@@ -5,26 +5,43 @@
 import tempfile
 import time
 
+from .exc import WaitTimeout
+
 logger = logging.getLogger(__name__)
 
 TEST_SESSION_PREFIX = "libtmux_"
 RETRY_TIMEOUT_SECONDS = int(os.getenv("RETRY_TIMEOUT_SECONDS", 8))
+RETRY_INTERVAL_SECONDS = float(os.getenv("RETRY_INTERVAL_SECONDS", 0.05))
 
 namer = tempfile._RandomNameSequence()
 current_dir = os.path.abspath(os.path.dirname(__file__))
 example_dir = os.path.abspath(os.path.join(current_dir, "..", "examples"))
 fixtures_dir = os.path.realpath(os.path.join(current_dir, "fixtures"))
 
 
-def retry(seconds=RETRY_TIMEOUT_SECONDS):
+def retry(
```

Comment:
Can we:

- keep the old function (so we can deprecate it over the next release)
- rename this new function something like: `until`, `wait`, `retry`, (any other name you find suitable will do)

thoughts?

Reply:
sure thing